dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/datavalue/DataValue.java

```diff
@@ -97,6 +99,14 @@
     @JacksonXmlProperty( isAttribute = true )
     private Boolean deleted;
 
+    @JsonProperty
+    @JacksonXmlProperty( isAttribute = true )
+    private String categoryCombo;
+
+    @JsonProperty
+    @JacksonXmlProperty
+    private Set<String> attributeCategoryOptions;
```

Comment:
For the new data value app, we need the "cat combo and cat options" approach both for disaggregation and attribute categories. So it would be good to design a model which supports both right away.

Reply:
this change has been removed from this PR